platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/graph/conflicts/DefaultCapabilitiesConflictHandler.java

```diff
@@ -289,8 +290,28 @@ private CapabilityConflict(String group, String name, Set<NodeState> nodes) {
                 }
             }
             this.descriptors = builder.build();
+
+            nodeToDependentNodes = buildDependentRelationships(nodes);
         }
 
+        private Map<NodeState, Set<NodeState>> buildDependentRelationships(Set<NodeState> nodes) {
```

Comment:
While this shows that accounting for this information solves the issue, I have concern around its performance impact.

I am looking into doing this processing only the second time we see a given conflict.

Reply:
Delayed computation of that information has been added